backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt

```diff
@@ -695,6 +695,30 @@ object Sessionize {
         )
     }
 
+    suspend fun importDevFestVenice2024(): Int {
+        return writeData(
+            getData("https://sessionize.com/api/v2/r14kt8fj/view/All"),
+            config = DConfig(
+                id = ConferenceId.DevFestVenice2023.id,
```

Comment:
Woops, I missed that one, this just overwrote the 2023 data. I'll make a patch quickly

Reply:
Should be all good now

Reply:
ups

Reply:
shall I re-push?

Reply:
Do the `curl -X post https://confetti-app.dev/update/devfestvenice2024` dance? I did it already, you should have data now.